framework/auth/core.py

```diff
@@ -1004,6 +1022,26 @@ def display_absolute_url(self):
     def deep_url(self):
         return '/profile/{}/'.format(self._primary_key)
 
+    @property
+    def get_unconfirmed_emails(self):
```

Comment:
Note: renaming to `unconfirmed_emails` isn't entirely accurate, since this returns a list of dicts, not a list of email addresses. Maybe `unconfirmed_email_info`?